data/Ghana/Parliament/sources/instructions.json

```diff
@@ -11,7 +11,7 @@
       "type": "membership"
     },
     {
-      "file": "manual/members-term-6.csv",
+      "file": "manual/official-term-6.csv",
```

Comment:
If you're going to move this, I suspect you should be moving it to an `archive/` directory